container_registry/container_analysis/poll_discovery_finished.go

```diff
@@ -20,10 +20,10 @@ import (
 	"context"
 	"fmt"
 	"time"
+	"google.golang.org/api/iterator"
```

Comment:
Nit: would you mind moving this down one line with the other non-standard library imports?